System.Device.I2s/I2sBitsPerSample.cs

```diff
@@ -6,29 +6,28 @@
 namespace System.Device.I2s
 {
     /// <summary>
-    /// Defines how bit per sample is synchronized between devices on a I2s bus.
+    ///     Defines how bit per sample is synchronized between devices on a I2s bus.
```

Comment:
These empty spaces should be removed.

Reply:
Even for the "para" blocks or the "list" blocks where indentation would maybe beneficial for readability?

Reply:
Yes, just remove them all please. 

That breaks the style cop rules (which we don't have in this repo but will have in the future) and those will also show in the output docs that are generated from these, thus looking bad on the docs website.